cypress-tests/cypress/e2e/PaymentUtils/ItauBank.js

```diff
@@ -1,68 +1,67 @@
 export const connectorDetails = {
-    card_pm: {
-        ZeroAuthMandate: {
-            Response: {
-              status: 500,
-              body: {
-                error: {
-                    type: "invalid_request",
-                    message: "setup mandate flow not supported",
-                    code: "IR_20",
-                  },
-              },
-            },
+  card_pm: {
+    ZeroAuthMandate: {
+      Response: {
+        status: 500,
```

Comment:
Is this intentional? 4xx is expected here if I'm not wrong. Also, this can be picked from defaults which is in `commons.js`